cli-commands/command.js

```diff
@@ -23,6 +23,23 @@ class Command {
 
     execute(args) { }
 
+    static defineSubcommands(command, subcommands) {
```

Comment:
Implement one function `define`. It should define subcommands and options

Reply:
updated